<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="camera_combo"/>
         </item>
         <item>
          <widget class="QCheckBox" name="live_check">
           <property name="text">
            <string>Live</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="frame_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Exposure</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="exposure_spin">
           <property name="maximum">
            <number>1000000</number>
           </property>
           <property name="singleStep">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="exposure_slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="srgb_check">
           <property name="text">
            <string>sRGB</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="autosave_check">
           <property name="text">
            <string>Autosave</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_img_btn">
           <property name="text">
            <string>Save Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_h5_btn">
           <property name="text">
            <string>Save HDF5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Saved positions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="target_check">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Show target</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="pos_table">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>75</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Z</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="load_btn">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_btn">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="write_btn">
           <property name="toolTip">
            <string>To save to a new entry, deselect first by Ctrl + clicking</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QCheckBox" name="contrast_check">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Use 1 − x^(1/γ) instead of x^γ</string>
           </property>
           <property name="text">
            <string>Inverse γ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="auto_clim_check">
           <property name="text">
            <string>Auto color limits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="aspect_check">
           <property name="text">
            <string>Lock aspect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="ColorMapComboBox" name="cmap_combo"/>
         </item>
         <item>
          <widget class="ColorMapGammaWidget" name="gamma_widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="GraphicsLayoutWidget" name="graphics_layout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionsettings"/>
    <addaction name="separator"/>
    <addaction name="actioncrosshair"/>
    <addaction name="actioncircle"/>
    <addaction name="actioninvert"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsettings">
   <property name="text">
    <string>Open Settings</string>
   </property>
  </action>
  <action name="actioncrosshair">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Crosshair</string>
   </property>
  </action>
  <action name="actioncircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Circle</string>
   </property>
  </action>
  <action name="actioninvert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reverse Colors</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ColorMapComboBox</class>
   <extends>QComboBox</extends>
   <header>qt_extensions.colors</header>
  </customwidget>
  <customwidget>
   <class>ColorMapGammaWidget</class>
   <extends>QWidget</extends>
   <header>qt_extensions.colors</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exposure_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposure_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposure_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposure_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
